Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:16.04 as build
+
+MAINTAINER SDF Ops Team <ops@stellar.org>
```

Comment:
I think it should be your team rather than Ops? Not a big deal tough, can be fixed later.

Reply:
Cool that sounds good to me. I used the [stellar.org-config](https://github.com/stellar/stellar.org-config/blob/master/nginx/nginx.conf) config as a starting point, I left this alone. I can put my email.